Practice.scala

```diff
@@ -0,0 +1,21 @@
+package pioneer.practice
+
+object Practice {
+  /** given a number `n`, returns n^2^ */
+  def square(n: Int): Int = n * n
+
+  /**
+    * given a string `s`, returns a new string with
+    * two copies of `s`, back-to-back
+    */
+  def repeat(s: String): String = s + s
+
+  /**
+    * from http://codingbat.com/prob/p187868
+    *
+    * Returns true if we sleep in. We sleep in if it's not a weekday or
+    * we're on vacation.
+    */
+  def sleepIn(w: Boolean, v: Boolean): Boolean = w == false || v == true
+  // replace this comment with your implementation of sleepIn
+}
```

Comment:
It looks like this file was in the top directory of the repository. It's redundant here (there's already a version of this file in `/src/main/scala/pioneer/practice`.